pyqtgraph/functions.py

```diff
@@ -998,10 +999,10 @@ def solveBilinearTransform(points1, points2):
     
     return matrix
     
-def rescaleData(data, scale, offset, dtype=None, clip=None):
+def rescaleData(data, scale, offset, dtype=None, clip=None, log=False):
```

Comment:
Oof; `log` can also refer to logging. I think renaming it to something more specific would be helpful. `logScale`, `log10`, `logarithmic` come to mind.